src/Utility/Utils.cs

```diff
@@ -1,102 +1,128 @@
-                        else if (paramType == typeof(Stream))
-                        {
-                            // Stream is expected, so convert the byte array to a MemoryStream.
-                            value = new MemoryStream(blobBytes);
-                        }
-                    }
-
-                    // When the input is 'String', we don't attempt to convert it to bytes because if the blob
-                    // is in fact a binary file, it's impossible to get the right bytes back even if we use the
-                    // same encoding used by the host when converting those bytes to string.
-
-                    break;
-                default:
-                    break;
-            }
-
-            return value;
-        }
-
-        /// <summary>
-        /// Helper method to do additional transformation on the output value.
-        /// </summary>
-        internal static object TransformOutBindingValueAsNeeded(string bindingName, ReadOnlyBindingInfo bindingInfo, object value)
-        {
-            switch (bindingInfo.Type)
-            {
-                case "http":
-                    // Try converting the value to HttpResponseContext if it's not already an object of such type.
-                    if (value is HttpResponseContext)
-                    {
-                        break;
-                    }
-
-                    try
-                    {
-                        value = LanguagePrimitives.ConvertTo<HttpResponseContext>(value);
-                    }
-                    catch (PSInvalidCastException ex)
-                    {
-                        string errorMsg = string.Format(PowerShellWorkerStrings.FailToConvertToHttpResponseContext, bindingName, ex.Message);
-                        throw new InvalidOperationException(errorMsg);
-                    }
-
-                    break;
-                default:
-                    break;
-            }
-
-            return value;
-        }
-    }
-}
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using System.Text;
+using System.Threading;
+using Microsoft.PowerShell.Commands;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Utility
+{
+    internal class Utils
+    {
+        internal static CmdletInfo ImportModuleCmdletInfo = new CmdletInfo("Import-Module", typeof(ImportModuleCommand));
+        internal static CmdletInfo RemoveModuleCmdletInfo = new CmdletInfo("Remove-Module", typeof(RemoveModuleCommand));
+        internal static CmdletInfo GetJobCmdletInfo = new CmdletInfo("Get-Job", typeof(GetJobCommand));
+        internal static CmdletInfo RemoveJobCmdletInfo = new CmdletInfo("Remove-Job", typeof(RemoveJobCommand));
+
+        internal static Lazy<InitialSessionState> SingletonISS
```

Comment:
Please make this readonly otherwise it would be prone to get modified which is not the desired behavior

Reply:
Already addressed in my last commit.